C-Plus-Plus/cp/search_insert_position.cpp

```diff
@@ -0,0 +1,44 @@
+/*
+Given a sorted array and a target value, return the index if the target is found. If not, return the index where it would be if it were inserted in order.
+
+You may assume no duplicates in the array.
+*/
+#include <iostream>
+using namespace std;
+
+int search_insert_position(int arr[], int n, int target){
+    int l=0,r=n-1;
+    int mid;
+    while(l<=r){
+        mid = (l+r)/2;
```

Comment:
@bagladivyang03 Use `mid = l + ((h - l) / 2)` to avoid integer overflow condition.